components/PaymentMethodSelect.js

```diff
@@ -33,6 +33,7 @@ class PaymentMethodSelect extends Component {
 
     return (
       <StyledSelect
+        inputId="payment-method"
```

Comment:
We should avoid doing that since there can be multiple payment method pickers on the page. It's up to the places that use InputTypeCountry to define an `inputId`.